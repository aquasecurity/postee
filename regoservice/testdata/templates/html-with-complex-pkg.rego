package rego2.html
title:="Audit event received"
result:=sprintf("Audit event received from %s", [input.user])
url:="Audit-registry-received/Audit-image-received"