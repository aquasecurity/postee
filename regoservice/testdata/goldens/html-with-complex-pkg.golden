Audit event received from demo